docs/konnectors.md

```diff
@@ -208,6 +208,114 @@ single thread or application. But NaCl is [no longer
 maintained](https://bugs.chromium.org/p/chromium/issues/detail?id=239656#c160)
 and ZeroVM has some severe limitations, so, it won't be used.
 
+## Konnector isolation study
+
+The short list of tools which will be tested to isolate connectors is Rkt and NsJail which on paper better fullfill our needs.
+
+### NsJail
+
+NsJail is a lightweight process isolation tool, making use of Linux namespaces and seccomp-bpf syscall filters. It is not a container tool like docker. Its features are quiet extensive regarding isolation. The [README](https://github.com/google/nsjail) gives the full list of options available. Although available in the google github, it is not an official google tool.
+
+NsJail is:
+
+- easy to install : juste a make away with standard build tools
+- offers a full list or isolation tools
+- lightly documented the only documentation is nsjail -h (also available in the main github page) and it is quiet cryptic for a non-sysadmin         
+like me. I could not find any help in any search engine. Some examples are available to run a back in an isolated process and work but I could not run a full nodejs (only nodejs -v worked)
+- The konnectors will need a full nodejs installed on the host
+- Is still actively maintained
+
+### Rkt
+
+Rkt is very similar to docker. It can even directly run docker images from the docker registry, which gives us a lot of existing images to use, even if we want to be able to use other languages than node. For example, we could have also a container dedicated to weboob, another container could use phantomjs or casper and without forcing self-hosted users to do complicated installation procedures.
+
+Rkt is :
+- easy to install : debian, rpm package available, archlinux community package : https://github.com/coreos/rkt/releases
+- has network isolation like docker
+- offers CPU, memory limitation, seccomp isolation (but the set of rules to use is out of my understanding)
+- is well [documented](https://coreos.com/rkt/docs/latest/), complete man pages, but not as well known as docker, then there is not a lot of things to find outside the official documentation.
+- can use docker image directly or can convert them to one runnable aci file with one simple cli command (rkt export)
+- is in active developpement but relatively stable regarding core features.
+- container images can be easily signed and the signature is checked by default when running a container.
+
+I managed to run a nodejs container with just the following commands :
+
+    rkt run --insecure-options=image --interactive docker://node:slim --name nodeslim -- -v
```

Comment:
Yes, if they belong to the rkt group. I will add the information in the document